splinterd/src/main.rs

```diff
@@ -502,164 +395,70 @@ fn get_transport(
                         server_cert,
                     )?;
 
-                    return Ok((Box::new(transport), log_value));
+                    return Ok((Box::new(transport), true));
                 }
             }
 
-            let cert_location = {
-                if let Ok(s) = env::var(CERT_DIR_ENV) {
-                    s
-                } else {
-                    DEFAULT_CERT_DIR.to_string()
-                }
-            };
+            let client_cert = config.client_cert();
+            if !Path::new(&client_cert).is_file() {
+                return Err(GetTransportError::CertError(
+                    "Must provide a valid client certificate".into(),
```

Comment:
These errors messages should also log the file that is being tested.

Reply:
Addressed.